app/models.py

```diff
@@ -1,105 +1,157 @@
 """
 This files handles all the database logic and instances
 """
-import itertools
+from datetime import datetime, timedelta
+import jwt
 from passlib.apps import custom_app_context as pwd_context
 
-from app import db
+from app import db, app
 
-class Users:
+#association table that associates userstable to events table
+rsvp = db.Table('rsvps',
+                db.Column('user_id', db.Integer, db.ForeignKey('user_db.id')),
+                db.Column('event_id', db.Integer,
+                          db.ForeignKey('events_db.id'))
+               )
+class Users(db.Model):
     """
     This class handles all the logic and methods
     associated with a user
     """
-    user_db = {}
-    id_generator = itertools.count(1)
+
+    __tablename__ = 'user_db'
+
+    id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(256), nullable=False, unique=True)
+    password = db.Column(db.String(256), nullable=False)
```

Comment:
consider hashing the user password and storing the hash rather than the original string

Reply:
ill refactor that